sdk/web-pubsub/web-pubsub/review/web-pubsub.api.md

```diff
@@ -0,0 +1,153 @@
+    excludedConnections?: string[];
+}
+
+// @public
+export interface HubSendToConnectionOptions extends OperationOptions {
+}
+
+// @public
+export interface HubSendToUserOptions extends OperationOptions {
+}
+
+// @public
+export class WebPubSubGroup {
+    // @internal
+    constructor(hubName: string, groupName: string);
+    addConnection(connectionId: string, options?: GroupAddConnectionOptions): Promise<RestResponse>;
+    addUser(username: string, options?: GroupAddUserOptions): Promise<RestResponse>;
+    readonly apiVersion: string;
+    endpoint: string;
+    readonly groupName: string;
+    hasUser(username: string, options?: GroupHasUserOptions): Promise<boolean>;
+    readonly hubName: string;
+    removeConnection(connectionId: string, options?: GroupRemoveConnectionOptions): Promise<RestResponse>;
+    removeUser(username: string, options?: GroupRemoveUserOptions): Promise<RestResponse>;
+    // Warning: (ae-forgotten-export) The symbol "GroupSendTextToAllOptions" needs to be exported by the entry point index.d.ts
+    sendToAll(message: string, options: GroupSendTextToAllOptions): Promise<RestResponse>;
+    // Warning: (ae-forgotten-export) The symbol "JSONTypes" needs to be exported by the entry point index.d.ts
+    // Warning: (ae-forgotten-export) The symbol "GroupSendToAllOptions" needs to be exported by the entry point index.d.ts
+    sendToAll(message: JSONTypes, options?: GroupSendToAllOptions): Promise<RestResponse>;
+    sendToAll(message: HttpRequestBody, options?: GroupSendToAllOptions): Promise<RestResponse>;
+}
+
+// @public
+export class WebPubSubServiceClient {
+    constructor(connectionString: string, hubName: string, options?: HubAdminClientOptions);
+    constructor(endpoint: string, credential: AzureKeyCredential, hubName: string, options?: HubAdminClientOptions);
+    readonly apiVersion: string;
+    closeConnection(connectionId: string, options?: CloseConnectionOptions): Promise<RestResponse>;
+    endpoint: string;
+    getAuthenticationToken(options?: GetAuthenticationTokenOptions): Promise<GetAuthenticationTokenResponse>;
+    // Warning: (ae-forgotten-export) The symbol "Permission" needs to be exported by the entry point index.d.ts
+    // Warning: (ae-forgotten-export) The symbol "HubGrantPermissionOptions" needs to be exported by the entry point index.d.ts
+    //
+    // (undocumented)
+    grantPermission(connectionId: string, permission: Permission, options?: HubGrantPermissionOptions): Promise<RestResponse>;
+    group(groupName: string): WebPubSubGroup;
+    hasConnection(connectionId: string, options?: HasConnectionOptions): Promise<boolean>;
+    hasGroup(groupName: string, options?: HubHasGroupOptions): Promise<boolean>;
+    hasUser(username: string, options?: HubHasUserOptions): Promise<boolean>;
+    readonly hubName: string;
+    removeUserFromAllGroups(userId: string, options?: CloseConnectionOptions): Promise<RestResponse>;
```

Comment:
Does this remove the user from the hub as well?

Reply:
It does not.